_infra/helm/securebanking-openbanking-uk-rs/values.yaml

```diff
@@ -1,40 +1,47 @@
+---
+configmap:
+  apiVersion: v1
+  # Addition Spring properties can be provided as a yaml block here, this will be mounted into the container and override / extend the config found in the application.yml inside the jar
```

Comment:
Where can we find documentation of these spring properties and what they do, why we might want to change them etc?

Reply:
I'm not actually sure on the info on this, an example can be found [here](https://github.com/ForgeCloud/secure-api-gateway-relenv-deployments/blob/main/ob-sandbox-v1/values.yaml#L70) for relenvs, and for devenvs [here](https://github.com/SecureApiGateway/sbat-cd/blob/master/argo/apps/templates/test-facility-bank.yaml#L24)